src/dbnode/storage/block/lease.go

```diff
@@ -181,13 +180,6 @@ func (m *leaseManager) UpdateOpenLeases(
 func (m *leaseManager) SetLeaseVerifier(leaseVerifier LeaseVerifier) error {
 	m.Lock()
 	defer m.Unlock()
-
-	if m.verifier != nil {
```

Comment:
Just from curiosity what's the motivation to remove this check?

Reply:
It needs to be called multiple times so that integration tests can open and close the database multiple times. The check was overly paranoid anyways so it was just easiest to remove it.